nifi-commons/nifi-web-client-api/src/main/java/org/apache/nifi/web/client/api/HttpResponseEntity.java

```diff
@@ -43,4 +44,11 @@ public interface HttpResponseEntity extends Closeable {
      * @return HTTP Response Body stream can be empty
      */
     InputStream body();
+
+    /**
+     * Get the HTTP request endpoint that was accessed to generate this HTTP
+     *
+     * @return the HTTP request endpoint
+     */
+    URI getRequestEndpoint();
```

Comment:
I recommend naming this `requestUri`, or simply or `uri`, similar to the [HttpResponse](https://docs.oracle.com/en/java/javase/21/docs/api/java.net.http/java/net/http/HttpResponse.html) model.
 ```suggestion
    URI getUri();
```